skimage/morphology/tests/test_rolling_ball.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tests for Rolling Ball Filter
+(skimage.morphology.rolling_ball)
+"""
+
+import numpy as np
+
+from skimage import data
+from skimage.morphology import rolling_ball
+from skimage.util import invert
+
+
+def test_white_background():
+    img = 23 * np.ones((100, 100), dtype=np.uint8)
+    img = invert(img)
+    result = rolling_ball(img, radius=1, white_background=True)
+    assert np.allclose(result, 255 * np.ones_like(img))
+
+
+def test_black_background():
+    img = 23 * np.ones((100, 100), dtype=np.uint8)
+    result = rolling_ball(img, radius=1)
+    assert np.allclose(result, np.zeros_like(img))
+
+
+def test_float_img():
+    img = data.coins().astype(np.float32)
```

Comment:
Please, run this test by using a context manager. `with testing.raises(ValueError):`

Reply:
This test is no longer necessary because float images are now valid.